ui/src/components/song/SongEditorContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import SongEditorDesktopView from './SongEditorDesktopView';
+import SongEditorMobileView from './SongEditorMobileView';
+
+interface windowSize {
+  width: number | undefined;
+  height: number | undefined;
+}
+
+// Custom Hook to detect screen size
+function useWindowSize() {
```

Comment:
i think this can be simplified

const isMobile = useMediaQuery('(max-width: 768px)');
const SongEditorView= isMobile ? <<SongEditorMobileView actionOnEditor="ADD NEW"  /> : <SongEditorDesktopView actionOnEditor="ADD NEW" />

{SongEditorView}